R/create.R

```diff
@@ -85,14 +85,11 @@ create_project <- function(path,
 
 #' Create a repo and project from GitHub
 #'
-#' Creates a new local Git repository from a repository on GitHub. If you have
-#' pre-configured a GitHub personal access token (PAT) as described in
-#' [gh::gh_whoami()], you will get more sensible default behavior for the `fork`
-#' argument. You cannot create a fork without a PAT. Currently only works for
-#' public repositories. A future version of this function will likely have an
-#' interface closer to [use_github()], i.e. more ability to accept credentials
-#' and more control over the Git configuration of the affected remote or local
-#' repositories.
+#' Creates a new local Git repository from a repository on GitHub. It is highly
+#' recommended that you pre-configure or pass a GitHub personal access token
+#' (PAT) as described in [gh::gh_whoami()]. In particular, a PAT is required in
```

Comment:
Are you planning to add any PAT help for this release?

Reply:
Yes, I plan to tackle that next.